core/pkg/evaluator/json.go

```diff
@@ -400,16 +400,36 @@ func getFlagdProperties(context map[string]any) (flagdProperties, bool) {
 	return p, true
 }
 
+func (je *JSON) loadAndCompileSchema() *gojsonschema.Schema {
+	schemaLoader := gojsonschema.NewSchemaLoader()
+
+	targetingSchemaLoader := gojsonschema.NewStringLoader(schema.Targeting)
+	if err := schemaLoader.AddSchemas(targetingSchemaLoader); err != nil {
+		je.Logger.Warn(fmt.Sprintf("error adding Targeting schema: %s", err))
+	}
+
+	flagdDefinitionsLoader := gojsonschema.NewStringLoader(schema.FlagdDefinitions)
```

Comment:
```suggestion
        // compile root schema
	flagdDefinitionsLoader := gojsonschema.NewStringLoader(schema.FlagdDefinitions)
```

Suggesting this and above, since the schemaLoader method names don't really explain this nuance, IMHO. This might be helpful to people.